solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/inbound/InboundXMLMessageListener.java

```diff
@@ -61,6 +61,7 @@
 		this.attributesHolder = attributesHolder;
 		this.needHolder = needHolder;
 		this.needAttributes = needAttributes;
+		this.flowReceiverContainer.setXMLMessageMapper(xmlMessageMapper);
```

Comment:
Minor suggestion: I think the `xmlMessageMapper` instantiation can be moved to where the `FlowReceiverContainer` is instantiated. That way `FlowReceiverContainer` can have its `xmlMessageMapper` be a `final` field, and we wouldn't need to deal with a potentially null `xmlMessageMapper`.